Example/Sources/ViewControllers/WebViewController.swift

```diff
@@ -40,55 +40,55 @@ final class WebViewController: UIViewController {
       )
     }
   }
-
-
+  
+  
   // MARK: Layout
-
+  
   override func viewDidLayoutSubviews() {
     super.viewDidLayoutSubviews()
     self.webView.frame = self.view.bounds
     self.activityIndicatorView.center.x = self.view.frame.width / 2
     self.activityIndicatorView.center.y = self.view.frame.height / 2
   }
-
-
+  
+  
   // MARK: Actions
-
+  
   dynamic func doneButtonDidTap() {
     self.dismiss(animated: true, completion: nil)
   }
-
+  
 }
 
 
 // MARK: - URLNavigable
 
 extension WebViewController: URLNavigable {
-
-  convenience init?(url: URLConvertible, values: [String: Any]) {
+  
+  convenience init?(url: URLConvertible, values: [String: Any], userInfo: [AnyHashable: Any]?) {
```

Comment:
About this, because, we can't set default argument in initialize method. Do you have some mind about this.